server/src/main/java/app/user/UserProfileResponse.java

```diff
@@ -20,6 +20,7 @@ public record UserProfileResponse(
         String imageUrl,
 
         @Schema(requiredMode = Schema.RequiredMode.REQUIRED)
-        LocalDateTime dateJoined
-) {}
+        Date dateJoined
```

Comment:
Date is an outdated library. In modern java
LocalDateTime 
and
ZonedDateTime are used
LocalDateTime is 12 dec 14:00 2024 
ZonedDateTime is  12 dec 14:00 2024 WARSZAWA
So it has Zone in it. 

Why it is better?
2 reasons 
1) ZonedDateTime has a lot of cool utilities so you can transfomr time freely , that Date doesnt have
2) Date has some bugs when dealing with time changes for summer/winter seazons in time when you have +1  -1 hour 

So just uze ZonedDateTime instead of Date